apps/remix-ide-e2e/src/commands/addAtAddressInstance.ts

```diff
@@ -1,8 +1,9 @@
-const EventEmitter = require('events')
+import { NightwatchBrowser } from 'nightwatch'
+import EventEmitter from "events"
 
 class addAtAddressInstance extends EventEmitter {
-  command (address, isValidFormat, isValidChecksum) {
-    this.api.perform((done) => {
+  command (this: NightwatchBrowser, address: string, isValidFormat: boolean, isValidChecksum: boolean): NightwatchBrowser {
```

Comment:
sounds like risky to name a variable `this`, could be addressed in an another PR..

Reply:
It is not a variable. `this` is the context which the command function uses. It is normally passed by the caller of the `command` function from nightwatch. Also, passing parameters into the `command` function without specifying `this` works perfectly. It just uses the default context for `this` which is the `addAtAddressInstance` class for this example.